acquia/acquia_cms
# ACMS-1468: Drupal 10 Readiness - Update contrib version in ACMS Site Studio, Image, Search Module.

**Motivation**
Fixes #ACMS-1468

**Proposed changes**
Drupal 10 Readiness - Update contrib version in ACMS Image, Search, Site Studio Module.

**Alternatives considered**
NA

**Testing steps**

- Install site with this code `/vendor/bin/acms site:install`.
- Verify everything works fine.

**Merge requirements**
